--- a/src/defaults.qtml
+++ b/src/defaults.qtml
@@ -78,21 +78,21 @@
     <atom element="Mn" accuracy="Low"    pao="Mn6.0"  mode="s2p1d1"  vps="Mn_CA13" />
     <atom element="Mn" accuracy="Normal" pao="Mn8.0"  mode="s3p2d2f1"  vps="Mn_CA13" />
     <atom element="Mn" accuracy="High"   pao="Mn10.0"  mode="s3p3d3f1"  vps="Mn_CA13" />
-    <atom element="Fe" accuracy="Low"    pao="Fe6.0S"  mode="s1p1d1"  vps="Fe_CA13" />
-    <atom element="Fe" accuracy="Normal" pao="Fe8.0S"  mode="s2p2d2f1"  vps="Fe_CA13" />
-    <atom element="Fe" accuracy="High"   pao="Fe10.0S"  mode="s3p3d2"  vps="Fe_CA13" />
-    <atom element="Co" accuracy="Low"    pao="Co6.0S"  mode="s1p1d1"  vps="Co_CA13" />
-    <atom element="Co" accuracy="Normal" pao="Co8.0S"  mode="s2p2d2f1"  vps="Co_CA13" />
-    <atom element="Co" accuracy="High"   pao="Co10.0S"  mode="s3p4d3f2"  vps="Co_CA13" />
-    <atom element="Ni" accuracy="Low"    pao="Ni6.0S"  mode="s1p1d1"  vps="Ni_CA13" />
-    <atom element="Ni" accuracy="Normal" pao="Ni8.0S"  mode="s2p2d2f1"  vps="Ni_CA13" />
-    <atom element="Ni" accuracy="High"   pao="Ni10.0S"  mode="s4p4d3f2"  vps="Ni_CA13" />
-    <atom element="Cu" accuracy="Low"    pao="Cu6.0S"  mode="s1p1d1"  vps="Cu_CA13" />
-    <atom element="Cu" accuracy="Normal" pao="Cu8.0S"  mode="s2p2d2f1"  vps="Cu_CA13" />
-    <atom element="Cu" accuracy="High"   pao="Cu10.0S"  mode="s3p3d3"  vps="Cu_CA13" />
-    <atom element="Zn" accuracy="Low"    pao="Zn6.0S"  mode="s1p1d1"  vps="Zn_CA13" />
-    <atom element="Zn" accuracy="Normal" pao="Zn8.0S"  mode="s2p2d2f1"  vps="Zn_CA13" />
-    <atom element="Zn" accuracy="High"   pao="Zn10.0S"  mode="s3p3d2f2"  vps="Zn_CA13" />
+    <atom element="Fe" accuracy="Low"    pao="Fe6.0S"  mode="s1p1d1"  vps="Fe_CA13S" />
+    <atom element="Fe" accuracy="Normal" pao="Fe8.0S"  mode="s2p2d2f1"  vps="Fe_CA13S" />
+    <atom element="Fe" accuracy="High"   pao="Fe10.0S"  mode="s3p3d2"  vps="Fe_CA13S" />
+    <atom element="Co" accuracy="Low"    pao="Co6.0S"  mode="s1p1d1"  vps="Co_CA13S" />
+    <atom element="Co" accuracy="Normal" pao="Co8.0S"  mode="s2p2d2f1"  vps="Co_CA13S" />
+    <atom element="Co" accuracy="High"   pao="Co10.0S"  mode="s3p4d3f2"  vps="Co_CA13S" />
+    <atom element="Ni" accuracy="Low"    pao="Ni6.0S"  mode="s1p1d1"  vps="Ni_CA13S" />
+    <atom element="Ni" accuracy="Normal" pao="Ni8.0S"  mode="s2p2d2f1"  vps="Ni_CA13S" />
+    <atom element="Ni" accuracy="High"   pao="Ni10.0S"  mode="s4p4d3f2"  vps="Ni_CA13S" />
+    <atom element="Cu" accuracy="Low"    pao="Cu6.0S"  mode="s1p1d1"  vps="Cu_CA13S" />
+    <atom element="Cu" accuracy="Normal" pao="Cu8.0S"  mode="s2p2d2f1"  vps="Cu_CA13S" />
+    <atom element="Cu" accuracy="High"   pao="Cu10.0S"  mode="s3p3d3"  vps="Cu_CA13S" />
+    <atom element="Zn" accuracy="Low"    pao="Zn6.0S"  mode="s1p1d1"  vps="Zn_CA13S" />
+    <atom element="Zn" accuracy="Normal" pao="Zn8.0S"  mode="s2p2d2f1"  vps="Zn_CA13S" />
+    <atom element="Zn" accuracy="High"   pao="Zn10.0S"  mode="s3p3d2f2"  vps="Zn_CA13S" />
     <atom element="Ga" accuracy="Low"    pao="Ga7.0"  mode="s1p1d1"  vps="Ga_CA13" />
     <atom element="Ga" accuracy="Normal" pao="Ga8.0"  mode="s2p2d2f1"  vps="Ga_CA13" />
     <atom element="Ga" accuracy="High"   pao="Ga9.0"  mode="s3p3d3f1"  vps="Ga_CA13" />
--- a/src/symbol.h
+++ b/src/symbol.h
@@ -51,7 +51,7 @@
   switch( number ){
   case H  : return 1;
   case He : return 2;
-  case Li : return 1;
+  case Li : return 2+1;
   case Be : return 2;
   case B  : return 2+1;
   case C  : return 2+2;
@@ -70,13 +70,13 @@
   case K  : return 1;
   case Ca : return 2;
   case Sc : return 1+2;
-  case Ti : return 2+2;
+  case Ti : return 6+6;
   case V  : return 3+2;
   case Cr : return 5+1;
   case Mn : return 2+5;
-  case Fe : return 6+2;
-  case Co : return 2+7;
-  case Ni : return 8+2;
+  case Fe : return 10+4;
+  case Co : return 8+7;
+  case Ni : return 8+8;
   case Cu : return 10+1;
   case Zn : return 10+2;
   case Ga : return 2+1;
@@ -130,7 +130,7 @@
   case Au : return 10+1;
   case Hg : return 10+2;
   case Tl : return 2+1;
-  case Pb : return 2+2;
+  case Pb : return 7+7;
   case Bi : return 2+3;
   case Po : return 2+4; // undef
   case At : return 2+5; // undef
